.eslintrc

```diff
@@ -45,7 +45,7 @@
     "react/no-array-index-key": 0,
     "react/no-unused-prop-types": 0,
     "import/prefer-default-export": 0,
-    "import/no-extraneous-dependencies": ["error", {"devDependencies": ["**/__tests__/*.js"]}],
+    "import/no-extraneous-dependencies": ["error", {"devDependencies": ["**/__tests__/*.js", "**/__int-tests__/*.js"]}],
```

Comment:
Nit: I'd rename the `__int-test__` directory to `__appium-tests__` or even just `__tests__` (and let the fact that it's a top-level directory indicates that it's integration and not unit tests)